qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -814,11 +814,12 @@ def __init__(self, name, instrument, **kwargs):
             setpoint_units=(('V',),) * 2,
             **kwargs)
         self._instrument = instrument
-        self.data = _FMTResponse(None, None, None, None)
+
         self.param1 = _FMTResponse(None, None, None, None)
```

Comment:
I just learned that instance attributes can be documented like this https://github.com/QCoDeS/Qcodes/pull/2005/files#diff-7ab35e1b5092c8dd21f4bb9883f61a76R1094 should we document these 4?

Reply:
sure